docs/admin.rst

```diff
@@ -3,47 +3,52 @@ Admin Guide
 
 .. include:: ./includes/provide-feedback.txt
 
-You (the admin) should have your own username and passphrase, plus
-two-factor authentication through either the FreeOTP app
-on your smartphone or a YubiKey.
+The SecureDrop architecture contains multiple machines and hardened servers.
+While we have automated many of the installation and maintenance tasks, a
+skilled Linux admin is required to responsibly run the system.
+
+This section outlines the tasks the admin is responsible for in order to
+ensure that the SecureDrop server continues to be a safe place for sources to
+talk to journalists.
 
 .. _Responsibilities:
 
 Responsibilities
 ----------------
 
-The SecureDrop architecture contains multiple hardened servers, and while we have
-automated many of the installation and maintenance tasks, a skilled Linux
-admin and some manual intervention is required to responsibly run the system.
+Maintaining Credentials
+~~~~~~~~~~~~~~~~~~~~~~~
 
-This section outlines the tasks the admin is responsible for in order to
-ensure that the SecureDrop server continues to be a safe place for sources to
-talk to journalists.
+The admin should have her own username, passphrase, and two-factor
+authentication method (via smartphone app or YubiKey). Admins are also
+responsible for tracking user credentials and encouraging best practices. (See
+:doc:`passphrases` and :doc:`passphrase_best_practices`.)
 
-Keep Your SecureDrop Server Updated
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Updating the SecureDrop Servers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You should maintain awareness of SecureDrop updates and take any required
+The admin should be aware of all SecureDrop updates and take any required
 manual action if requested in the `SecureDrop Release Blog`_. We recommend
 subscribing to the `SecureDrop RSS Feed`_ to stay apprised of new updates.
 
 Most often, the SecureDrop server will automatically update via apt. However,
-occasionally you will need to run the Ansible playbooks. We will inform you in
+occasionally you will need to run Ansible playbooks. We will inform you in
```

Comment:
ah er oops, this is old. Bonus points for replacing "run Ansible playbooks" with "run ``securedrop-admin install``" (double backticks for fixed-width)